core/src/main/java/org/mskcc/cbio/portal/dao/DaoGenePanel.java

```diff
@@ -34,7 +34,10 @@
 
 import java.sql.*;
 import java.util.*;
+import java.util.stream.Collectors;
```

Comment:
The existing methods in this file all interact with the database in some way. Lets move your utility methods into a seperate class in `core/src/main/java/org/mskcc/cbio/portal/util`. Put all methods that don't interact with the database directly there.

Reply:
just to clarify: are you referring to the union/intersection/difference methods that use streams?

Reply:
Just moved getPropertyValue and getGenes to GenePanelUtil.java (new file) in core\src\main\java\org\mskcc\cbio\portal\util